nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -91,6 +91,16 @@ BtActionServer<ActionT>::BtActionServer(
       RCLCPP_INFO_STREAM(logger_, "Error_code parameters were set to:" << error_codes_str);
     }
   }
+
+  // Result timeout for the action server.
+  // In https://github.com/ros2/rcl/pull/1012 a change was introduced
+  // which makes action servers discard a goal handle if the result is not produced
+  // within 10 seconds. Since this may not be the case for all actions in
+  // Nav2, this timeout is exposed as a parameter and defaults to the previous
+  // expiration value of 15 minutes.
+  if (!node->has_parameter("action_server_result_timeout")) {
+    node->declare_parameter("action_server_result_timeout", 15 * 60);
```

Comment:
```suggestion
    node->declare_parameter("action_server_result_timeout", 900.0);
```

Make it clearly a double since declaring the parameter also declares its type